packages/nextjs-kit/src/lib/sortChar.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * @param {Array<Record<string, string | number>>} obj object to sort
+ * @param {string} key to sort object by
+ * @param {string} direction to sort object by
+ * @returns {Array<Record<string, string | number>>} the sorted object that was passed in
+ * @remarks
+ * This is a function to sort any character object on a specific key in a direction of the users choice.
+ *
+ * Note:
+ * There is a named and default export so that we can
+ * support submodules for the ESModule build and a single entrypoint for the UMD build
+ */
+export const sortChar = (
+  obj: Array<Record<string, string | number>>,
+  key: string,
+  direction: string
```

Comment:
Can we make these parameters an object?
also here is a case where an `enum` may be a good idea for the `direction` type, also I would recommend shortening the names from ascending and descending to `'ASC'` and `'DESC'`.